app/graphql/types/board_type.rb

```diff
@@ -14,5 +14,10 @@ class BoardType < Types::BaseObject
     field :columns, [ColumnType], null: false
     field :cards, [CardType], null: false
     field :action_items, [ActionItemType], null: false
+    field :participants, [ParticipantType], null: false
+
+    def participants
+      Board::Participants.new(board_id: board_id).participant_ids
```

Comment:
Return a list of users, not ids.
```suggestion
      Board::Participants.new(board_id: board_id).users
```